pipeline/matcher/src/main/scala/uk/ac/wellcome/platform/matcher/storage/dynamo/DynamoBatchWriter.scala

```diff
@@ -18,22 +18,6 @@ class DynamoBatchWriter[T](config: DynamoConfig)(
     Future {
       val ops = table.putAll(items.toSet)
 
-      scanamo.exec(ops).foreach { result =>
```

Comment:
Why can we remove this now?

Reply:
Previously Scanamo exposed the underlying results for us (as a Seq[BatchWriteItemResult)); now it just returns a Unit.

I haven't checked in detail, but it looks like it's doing more work to inspect the results of a BatchWriteItemResult, try to write them, and would presumably return a failed Future if something went wrong: https://github.com/scanamo/scanamo/blob/1a1282da0e3e7068885feb5ff1f08e77bb95c37d/scanamo/src/main/scala/org/scanamo/ScanamoFree.scala#L51-L77